pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -168,10 +168,18 @@ def cmd(self):
         if self.verbosity < 1:
             cmd.append("--quiet")
         elif self.verbosity > 1:
-            cmd.append("--verbose")
+            # currently only two verbosity settings are supported, so using `-vvv`
+            # in place of `--verbose`, because it is needed to see migrations.
+            cmd.append("-vvv")
 
         if self.disable_capture:
             cmd.append("-s")
+
+        # TODO: Restore option
+        # if not self.disable_migrations:
+        if True:
```

Comment:
1. I made this `if True` temporarily to test on Jenkins.
2. I added reference to `disable_migrations` in this file, which was previously only used in https://github.com/edx/edx-platform/blob/9cf2f9f298e5e8be3b3abcaadaf0b7a96d0de0df/pavelib/utils/test/suites/python_suite.py#L23-L31.